aiida/backends/sqlalchemy/migrations/versions/1de112340b18_django_parity_3.py

```diff
@@ -0,0 +1,222 @@
+# -*- coding: utf-8 -*-
+###########################################################################
+# Copyright (c), The AiiDA team. All rights reserved.                     #
+# This file is part of the AiiDA code.                                    #
+#                                                                         #
+# The code is hosted on GitHub at https://github.com/aiidateam/aiida-core #
+# For further information on the license, see the LICENSE.txt file        #
+# For further information please visit http://www.aiida.net               #
+###########################################################################
+# pylint: disable=invalid-name,no-member
+"""Parity with Django backend (rev: 0049),
+part 3: Add PostgreSQL-specific indexes
+
+Revision ID: 1de112340b17
+Revises: 2b40c8131fe0
+Create Date: 2021-08-25 04:28:52.102767
+
+"""
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = '1de112340b18'
+down_revision = '1de112340b17'
+branch_labels = None
+depends_on = None
+
+# table name, column name, index name
+MISSING_STANDARD_INDEXES = (
+    ('db_dbauthinfo', ('aiidauser_id',), False, 'db_dbauthinfo_aiidauser_id_0684fdfb'),
+    ('db_dbauthinfo', ('dbcomputer_id',), False, 'db_dbauthinfo_dbcomputer_id_424f7ac4'),
+    ('db_dbcomment', ('dbnode_id',), False, 'db_dbcomment_dbnode_id_3b812b6b'),
+    ('db_dbcomment', ('user_id',), False, 'db_dbcomment_user_id_8ed5e360'),
+    ('db_dbgroup', ('user_id',), False, 'db_dbgroup_user_id_100f8a51'),
+    ('db_dblog', ('dbnode_id',), False, 'db_dblog_dbnode_id_da34b732'),
+    ('db_dbnode', ('ctime',), False, 'db_dbnode_ctime_71626ef5'),
+    ('db_dbnode', ('mtime',), False, 'db_dbnode_mtime_0554ea3d'),
+    ('db_dbnode', ('dbcomputer_id',), False, 'db_dbnode_dbcomputer_id_315372a3'),
+    ('db_dbnode', ('user_id',), False, 'db_dbnode_user_id_12e7aeaf'),
+)
+
+# table name, column name, index name
+MISSING_VARCHAR_INDEXES = (
+    ('db_dbcomputer', 'label', 'db_dbcomputer_label_bc480bab_like'),
+    ('db_dbgroup', 'label', 'db_dbgroup_name_66c75272_like'),
```

Comment:
Should this not be label after the Django migration 48 is performed correcting the migration name?
```suggestion
    ('db_dbgroup', 'label', 'db_dbgroup_label_66c75272_like'),
```

Why was this not caught by the tests?

Reply:
No this is from `aiida/backends/djsite/db/migrations/0021_dbgroup_name_to_label_type_to_type_string.py` (and there is also `aiida/backends/djsite/db/migrations/0030_dbnode_type_to_dbnode_node_type.py`), which obviously cannot be retroactively fixed.
I tempted then to add a django migration for these.

Reply:
Hang on. If its correct, maybe just leave it. Especially if we think that after Django is fully dropped, we might add a final SQLA migration to undo the things we had to do specifically for django, such as weirdly named indices and no server defaults. If we are planning on doing this, no need to change it for django now